google-cloud-bigtable/lib/google/cloud/bigtable/table.rb

```diff
@@ -238,8 +238,8 @@ def exists?
         # update, or delete operations.
         #
         # @param name [String] Name of the column family
-        # @param gc_rule [Google::Cloud::Bigtable::GcRule] Optional.
-        #   Required for create and update operations.
+        # @param gc_rule [Google::Cloud::Bigtable::GcRule, nil] A garbage
```

Comment:
I think listing `nil` here might be confusing. What do you think about something like this:

```ruby
# @param gc_rule [Google::Cloud::Bigtable::GcRule] The garbage collection
#   rule to be used for the column family. Optional. The service default value
#   will be used when not specified.
```

Reply:
I like it. I'll update with that wording everywhere.

Reply:
Updated.